tests/backtest/test_trade_request.py

```diff
@@ -0,0 +1,40 @@
+"""Test the TradeRequest class."""
+
+import pytest
+
+from coin_test.backtest import TradeRequest
+from coin_test.util import Side, TradeType
+
+
+def test_trade_request(example_symbol: str) -> None:
+    """Initialize correctly."""
+    example_side = Side.BUY
+    example_trade_type = TradeType.MARKET
+    example_notional = 1000.0
+
+    x = TradeRequest(example_symbol, example_side, example_trade_type, example_notional)
+
+    assert x.symbol == example_symbol
+    assert x.side == example_side
+    assert x.type_ == example_trade_type
+    assert x.notional == example_notional
+
+
+def test_bad_trade_request(example_symbol: str) -> None:
+    """Error on bad parameters."""
```

Comment:
nit: Should we specify which paramters are bad. ie. "on bad notional parameter"

Reply:
I've added detail to the docstring now